sdk/spring-experimental/spring-cloud-azure-native-configuration/pom.xml

```diff
@@ -144,6 +144,25 @@
           </rules>
         </configuration>
       </plugin>
+      <!-- We must generate an empty javadoc JAR file to publish to Maven Central because this package does not contain any public sources -->
```

Comment:
Is this required by maven central or release pipeline?

Reply:
Yeah, we added this to pass the Maven central rules of release packages.